schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/aep-hygiene-ops",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "AEP Hygiene Operation Request",
+  "description": "AEP Hygiene operation request that instructs AEP to deletes or modify records in the specified dataset and sandbox",
```

Comment:
"AEP Data Hygiene operations that instruct AEP to delete or modify records in a specified dataset and sandbox"

Reply:
Done